app/models/user.rb

```diff
@@ -88,6 +88,13 @@ def pretty_intent
     intent.tr('_', ' ')
   end
 
+  # Returns true if the user is the leader of the given group
+  #
+  # @return [Boolean]
+  def leader_of?(group)
```

Comment:
Ooo I feel like we could heavily refactor `GroupPolicy` and `DrawlessGroupPolicy`, if not `DrawPolicy. I'll open an issue for post-pilot.